cumulus_library/cli_parser.py

```diff
@@ -81,6 +75,26 @@ def add_aws_config(parser: argparse.ArgumentParser) -> None:
     )
 
 
+def add_db_config(parser: argparse.ArgumentParser) -> None:
+    """Adds arguments related to database backends to a subparser"""
+    group = parser.add_argument_group("Database config")
+    group.add_argument(
+        "--db-type",
+        help="Which database backend to use (default athena)",
+        choices=["athena", "duckdb"],
+        default="athena",
+    )
+    group.add_argument(
+        "--database",
+        # internally, we use PyAthena's terminology for this but the UX term is "database"
```

Comment:
we should probably touch this comment since it's not 100% accurate anymore.

Reply:
Edited to be more verbose about the situation.